README.rst

```diff
@@ -180,22 +185,75 @@ such as night mode or adjusting the subwoofer volume.
 
     $ songpal sound subwooferLevel 4
 
-Output Control
+
+Zone Control
 --------------
 
 .. code-block::
 
-    $ songpal output
+    $ songpal zone 
+
+    Zones:
+      * Main Zone (uri: extOutput:zone?zone=1) (active)
+      * Zone 2 (uri: extOutput:zone?zone=2) (active)
+      * Zone 3 (uri: extOutput:zone?zone=3)
+      * HDMI Zone (uri: extOutput:zone?zone=4)
+
+    $ songpal zone 'Main Zone'
+
+    Activating Main Zone (uri: extOutput:zone?zone=1) (active)
 
-    Outputs:
+    $ songpal zone 'Zone 2' true
+    
+    Activating Zone 2 (uri: extOutput:zone?zone=2) (active)
+
+    $ songpal zone 'Zone 2' false
+    
+    Deactivating Zone 2 (uri: extOutput:zone?zone=2) (active)
+
+Input Control
+--------------
+
+.. code-block::
+
+    $ songpal input
+
+    Inputs:
+      * SOURCE (uri: extInput:source)
+        - extOutput:zone?zone=2
+        - extOutput:zone?zone=3
+        - extOutput:zone?zone=4
+      * GAME (uri: extInput:game) (active)
+        - extOutput:zone?zone=1
+        - extOutput:zone?zone=4
+      * SAT/CATV (uri: extInput:sat-catv)
+        - extOutput:zone?zone=1
+        - extOutput:zone?zone=2
+        - extOutput:zone?zone=3
+        - extOutput:zone?zone=4
+      * VIDEO 1 (uri: extInput:video?port=1)
+        - extOutput:zone?zone=1
+        - extOutput:zone?zone=2
+        - extOutput:zone?zone=3
+        - extOutput:zone?zone=4
+      * VIDEO 2 (uri: extInput:video?port=2)
+        - extOutput:zone?zone=1
+        - extOutput:zone?zone=4
       * TV (uri: extInput:tv)
-      * HDMI1 (uri: extInput:hdmi?port=1)
-      * HDMI2 (uri: extInput:hdmi?port=2) (active)
-      * HDMI3 (uri: extInput:hdmi?port=3)
```

Comment:
Maybe keep these as an example when there is no zones available.

Reply:
Done